packages/js/components/src/search/index.tsx

```diff
@@ -45,113 +44,73 @@ type SearchType =
 	| 'variations'
 	| 'custom';
 
-type Props = {
-	type: SearchType;
+type State = {
+	options: Option[];
+};
+
+export type SearchProps = {
```

Comment:
I'd consider extracting these types to `./types.ts` to increase readability because there's quite lot of them in this file:
- `SearchType`
- `State`
- `SearchProps`

Reply:
done